src/aspire/utils/preprocess.py

```diff
@@ -292,16 +218,18 @@ def vol2img(volume, rots, L=None, dtype=None):
 
     im_f = Plan(volume.shape, -pts_rot).transform(volume)
 
-    im_f = m_reshape(im_f, (lv, lv, -1))
+    #im_f = m_reshape(im_f, (lv, lv, -1))
```

Comment:
Remove (also below).

Reply:
Oh geez, sorry, that was from debugging!